packages/jest-worker/src/Farm.ts

```diff
@@ -75,23 +75,17 @@ export default class Farm {
 
         if (computeWorkerKey) {
           hash = computeWorkerKey.call(this, method, ...args);
-          worker = hash == null ? null : this._cacheKeys[hash];
+          worker = hash === null ? null : this._cacheKeys[hash];
         }
 
         const onStart: OnStart = (worker: WorkerInterface) => {
-          if (hash != null) {
+          if (hash !== null) {
             this._cacheKeys[hash] = worker;
           }
         };
 
-        const onEnd: OnEnd = (error: Error | null, result: unknown) => {
-          customMessageListeners = [];
-          if (error) {
-            reject(error);
-          } else {
-            resolve(result);
-          }
-        };
+        const onEnd: OnEnd = (error: Error | null, result: unknown) =>
```

Comment:
Why are we not resetting `customMessageListeners = []` anymore?

Reply:
`customMessageListeners` wasn't being passed further, so I reckon it might be handled by js garbage collector itself?

Am I right?